example.js

```diff
@@ -1,30 +1,21 @@
-const { collector } = require("./build");
-const { join } = require("path");
+const { collector } = require('./build');
+const { join } = require('path');
 
 (async () => {
-  const EMULATE_DEVICE = "iPhone 13 Mini";
+    const URL = 'amazon.com';
+    const EMULATE_DEVICE = 'iPhone 13 Mini';
 
-  // Save the results to a folder
-  let OUT_DIR = true;
+    const config = {
+        inUrl: `http://${URL}`,
+        numPages: 3,
+        headless: true,
+        emulateDevice: EMULATE_DEVICE,
+        outDir: join(__dirname, 'demo-dir')
```

Comment:
Maybe add a comment here along the lines of `// optional, if omitted results will be returned but not saved to disk`, which I'm assuming is the point the original code was trying to get across? Unless that's not actually what happens if you omit it.

Reply:
It returns data either way. If you don't specify the directory, it creates a temporary directory for the logging library to write to, then deletes it at the end. This is something I want to clean up eventually.

```typescript
    if (outDir.includes('bl-tmp')) {
        clearDir(outDir, false);
    }
    return { status: 'success', ...output, reports };
};
```